src/shared/i18n/extension/empire/fr.ts

```diff
@@ -0,0 +1,311 @@
+import { LifeformDiscoveryEventArtifactFindingSize } from "@/shared/models/lifeform-discoveries/LifeformDiscoveryEventArtifactFindingSize";
+import { LifeformDiscoveryEventType } from "@/shared/models/lifeform-discoveries/LifeformDiscoveryEventType";
+import { EmpireTranslations } from "./type";
+import { RecursivePartial } from "@/shared/types/RecursivePartial";
+
+export const fr: RecursivePartial<EmpireTranslations> = {
+header: 'Empire',
+planet: 'Planète',
+production: {
+    header: 'Production',
+    subHeaders: {
+        resourceProduction: 'Production de ressources',
+        mineOverview: 'Vue d\'ensemble des mines',
+    },
+    averagePerHour: '⌀ par heure',
+    totalPerHour: 'total par heure',
+    totalPerDay: 'total par jour',
+    totalPerWeek: 'total par semaine',
+    activeProductionSettings: 'Paramètres de production actifs',
+    messageProduction100: 'Les valeurs affichées se réfèrent à un facteur de production de 100% et ne tiennent pas compte du manque d\'énergie.',
+
+    mines: {
+        crawlersAvailable: 'disponibles',
+        crawlersToReachLimit: count => `${count} foreuses pour le bonus maximum`,
+    },
+    items: 'Objets actifs',
+    breakdown: {
+        basicIncome: 'Revenu de base',
+        mineProduction: 'Production de mine',
+        consumption: 'Consommation',
+        lifeformBuildings: 'Bâtiments formes de vie',
+        crawlers: 'Foreuses',
+        items: 'Objets',
+        geologist: 'Géologue',
+        commandStaff: 'Conseil d\'officiers',
+        playerClass: 'Classe de joueur',
+        allianceClass: 'Classe d\'alliance',
+        lifeformTechnologies: 'Technologies formes de vie',
+    },
+
+    settings: {
+        reset: 'Réinitialiser',
+        header: 'Paramètres de production',
+        applyAndClose: 'Appliquer et fermer les paramètres de production',
+    },
+},
+amortization: {
+    header: 'Amortissement',
+
+    table: {
+        cost: 'Coût',
+        levels: 'Niveaux',
+        productionPlus: 'Production en plus',
+        amortizationTime: 'Temps d\'amortissement',
+        groupSelectedItems: 'Regrouper les objets sélectionnés',
+        showOriginalItems: 'Afficher tous les objets individuellement',
+
+        levelsOnPlanets: (levels, planets) => `${levels} ${levels == 1 ? 'niveau' : 'niveaux'} sur ${planets} ${planets == 1 ? 'planète' : 'planètes'}`,
+    },
+
+    settings: {
+        header: 'Paramètres',
+        applyAndClose: 'Appliquer les paramètres et fermer',
+
+        includeSettings: {
+            header: 'Inclure dans le calcul',
+
+            mines: 'Mines',
+            expeditions: 'Expéditions',
+            expeditionsRequirement: 'Remarque : Nécessite "Recherche des formes de vie" et/ou "Astrophysique" activée',
+            lifeformBuildings: ' Bâtiments des formes de vie ',
```

Comment:
the dev branch contains some changes to the amortization settings, so the keys `lifeformBuildings` and `lifeformTechnologies` have been removed and replaced by four more specific ones. Please take a look [here](https://github.com/ogametracker/ogame-extension/blob/dev/src/shared/i18n/extension/empire/en.ts#L69) for what changed